test/rest-api/request/body/body-text.test.ts

```diff
@@ -0,0 +1,54 @@
+import * as path from 'path'
+import { pageWith } from 'page-with'
+
+function prepareRuntime() {
+  return pageWith({
+    example: path.resolve(__dirname, 'body.mocks.ts'),
+  })
+}
+
+test('reads plain text request body as text', async () => {
+  const runtime = await prepareRuntime()
+  const res = await runtime.request('/text', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'text/plain',
+    },
+    body: 'hello-world',
+  })
+  const body = await res.text()
+
+  expect(res.status()).toBe(200)
+  expect(body).toBe('hello-world')
+})
+
+test('reads json request body as text', async () => {
```

Comment:
We should also add a case when you send nothing or explicit `null` as the request body, and assert what `req.text()` in the handler returns. I believe that must be an empty string.